api/src/test/java/com/epam/pipeline/dao/cluster/NodeDiskDaoTest.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.pipeline.dao.cluster;
+
+import com.epam.pipeline.AbstractSpringTest;
+import com.epam.pipeline.entity.cluster.DiskRegistrationRequest;
+import com.epam.pipeline.entity.cluster.NodeDisk;
+import com.epam.pipeline.entity.utils.DateUtils;
+import org.junit.After;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static com.epam.pipeline.util.CustomAssertions.assertThrows;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class NodeDiskDaoTest extends AbstractSpringTest {
+    
+    private static final String NODE_ID = "NODE_ID";
+    private static final String ANOTHER_NODE_ID = "ANOTHER_NODE_ID";
+    private static final String NULL_NODE_ID = null;
+    private static final Long SIZE = 1L;
+    private static final Long NULL_SIZE = null;
+
+    @Autowired
+    private NodeDiskDao dao;
+
+    @After
+    public void deleteDisksByNodeId() {
+        dao.deleteByNodeId(NODE_ID);
```

Comment:
You can just add `@Transactional` annotation to the class and all DB changes will be rolled-back.

Reply:
That's good. Thanks. Added.